server/app/views/components/FieldWithLabel.java

```diff
@@ -40,6 +40,8 @@ public class FieldWithLabel {
 
   protected OptionalLong minValue = OptionalLong.empty();
   protected OptionalLong maxValue = OptionalLong.empty();
+  private OptionalLong rows = OptionalLong.empty();
```

Comment:
Can you add the textarea comment above these, similar to the comment above for number fields.